src/order/spam-protection/spam-protection-actions.ts

```diff
@@ -0,0 +1,24 @@
+import { Action } from '@bigcommerce/data-store';
+
+export enum SpamProtectionActionType {
+    InitializeFailed = 'SPAM_PROTECTION_INITIALIZE_FAILED',
+    InitializeRequested = 'SPAM_PROTECTION_INITIALIZE_REQUESTED',
+    Completed = 'SPAM_PROTECTION_COMPLETED',
```

Comment:
Maybe for consistency with how we name actions, this should be:
`InitializeSucceeded = 'SPAM_PROTECTION_INITIALIZE_SUCCEEDED'`